curriculum/test/test-challenges.js

```diff
@@ -314,9 +314,22 @@ function populateTestsForLang({ lang, challenges, meta }) {
               throw new AssertionError(result.error);
             }
             const { id, title, block, dashedName } = challenge;
+            assert.exists(
+              dashedName,
+              `Missing dashedName for challenge ${id} in ${block}.`
+            );
             const pathAndTitle = `${block}/${dashedName}`;
-            mongoIds.check(id, title);
-            challengeTitles.check(title, pathAndTitle);
+            const idVerificationMessage = mongoIds.check(id, title);
+            assert.isNull(idVerificationMessage, idVerificationMessage);
+            const dupeTitleCheck = challengeTitles.check(
+              dashedName,
+              block,
+              pathAndTitle
+            );
```

Comment:
```suggestion
            const dupeTitleCheck = challengeTitles.check(dashedName, block);
```
We don't need to check `pathAndTitle` any more.

Reply:
Yeah, maybe someday.